osrf_testing_tools_cpp/test/cmake/test_osrf_testing_tools_cpp_filter_versions.cmake

```diff
@@ -21,24 +21,29 @@ if(valid_versions)
   message(FATAL_ERROR "expected valid versions to be empty, got '${valid_versions}'")
 endif()
 
-set(versions 1.7.0;1.8.0)
+set(versions 1.7.0;1.8.1)
```

Comment:
This may not be needed. This is a crafted test case that just happens to use the initial versions offered. I think if you undo this change then this commit https://github.com/osrf/osrf_testing_tools_cpp/pull/36/commits/aa230fdb015e3fa365af4170e1324f417792fff7 and maybe other changes can also be avoided.

This file is about testing just this filtering function, not the actual availability of googletest of one version or another. That kind of thing is actually tested here:

https://github.com/osrf/osrf_testing_tools_cpp/blob/2eefb7fc4d42b6243e24b83fa4f17faf5239faef/test_osrf_testing_tools_cpp/CMakeLists.txt#L17